### PR TITLE
Maestro dependency update without buildtools

### PR DESCRIPTION
**Pull request by ViktorHofer**
Relates to https://github.com/dotnet/coreclr/pull/20121

### CODE REVIEW

**Comment by jkotas** [comment]
Could you please disable the failing corefx test in https://github.com/dotnet/coreclr/blob/master/tests/CoreFX/CoreFX.issues.json to make the CI green?

**Comment by ViktorHofer** [comment]
Ok, done.

File: tests/CoreFX/CoreFX.issues.json [hunk 1]
```diff
@@ -407,4 +407,13 @@
             ]
         }
     }
+    {
```
Comment (tannergooding) [inline tests/CoreFX/CoreFX.issues.json]: Forgot a comma on L409

Comment (ViktorHofer) [reply tests/CoreFX/CoreFX.issues.json]: thanks, fixed.

**Comment by tannergooding** [comment]
@ViktorHofer, there is a build conflict in the issues.json, the run otherwise looks good.

**Comment by tannergooding** [comment]
@ViktorHofer, I rebased the changes in order to resolve the merge conflict.

**Pull request merged**